### ISSUE TITLE
[MASK] not added to special tokens by default

### ISSUE THREAD

**Issue by tomhosking** [huggingface/tokenizers#160]
Shouldn't `[MASK]` be added to special tokens by default for BERT?

The below doesn't work as I would have expected:
```
tokenizer = BertWordPieceTokenizer('./data/bert-vocabs/bert-base-uncased-vocab.txt', lowercase=True, add_special_tokens=True)

test = "This is a [MASK] sequence"
test_toks = tokenizer.encode(test)
[tokenizer.id_to_token(x) for x in test_toks.ids]
```
> ['[CLS]', 'this', 'is', 'a', '[', 'mask', ']', 'sequence', '[SEP]']

Calling `tokenizer.add_special_tokens(['[MASK]'])` before tokenizing does work as expected.

Related to #113

**Comment by julien-c** [comment]
I've noticed this as well with a GPT-2-like tokenizer for [this blog post](https://huggingface.co/blog/how-to-train), @n1t0 @mfuntowicz – so I believe it is a valid issue.

**Comment by n1t0** [comment]
Thank you for suggesting this @tomhosking! Fixed by 61b4c9c30a157aecea0ba633e83402944aaaa3e0
